lms/static/js/student_account/views/account_settings_fields.js

```diff
@@ -106,10 +108,11 @@
                     });
                 },
                 successMessage: function () {
-                    return this.indicators.success + StringUtils.interpolate(
+                    return HtmlUtils.joinHtml(
+                        this.indicators.success, StringUtils.interpolate(
```

Comment:
Put StringUtils on its own line.   (Note, this particular change happens to be personal preference).